src/http-middlewares/before/prepare-request.js

```diff
@@ -39,8 +39,7 @@ const coerceBody = (req) => {
 
   // auto coerce form if header says so
   if (contentType === FORM_TYPE && req.body && !_.isString(req.body)) {
-    // TODO: use const FormData = require('form-data'); instead?
-    req.body = querystring.stringify(req.body);
+    req.body = querystring.stringify(req.body).replace(/%20/g, '+');
```

Comment:
Sanity check that node doesn't have a built-in lib to do the encoding the way we want, vs having to do this find and replace?

Reply:
Nice idea! Unfortunately there isn't, because we want to do it for everything _but_ spaces. The [`options` for that method](https://nodejs.org/api/all.html#querystring_querystring_stringify_obj_sep_eq_options) only include sending a function that's `encodeURIComponent` by default.